config/config.go

```diff
@@ -1506,3 +1521,67 @@ func isValidCookieSize(maxCookieSize int) error {
 	}
 	return nil
 }
+
+type TmaxAdjustments struct {
+	// Flag indicating whether the tmax feature is enabled or not
+	Enabled bool `mapstructure:"enabled"`
+	// The maximum allowable tmax for the auction endpoint
+	// The tmax value used for the auction endpoint will be the minimum of the auction_max and tmax specified in the incoming request
+	AuctionMax int `mapstructure:"auction_max"`
+	// The maximum allowable tmax for the video endpoint
+	// The tmax value used for the video endpoint will be the minimum of the video_max and tmax specified in the incoming request
+	VideoMax int `mapstructure:"video_max"`
```

Comment:
What's your rationale behind adding separate upper bounds for auction, video and amp requests? I don't see a reason why we'd need these different upper bounds. We already have a common upper bound for all auction requests that is covered in the [AuctionTimeouts](https://github.com/prebid/prebid-server/blob/master/config/config.go#L41) config object.

Reply:
https://github.com/prebid/prebid-server/pull/2723/commits/eca34570c7d8c0a00f30c0fda9c32f0af25159c4 makes changes to remove upper bound from TmaxAdjustment config